### ISSUE TITLE
AdonisJS 6 Support

### ISSUE THREAD

**Issue by beletate** [RomainLanz/adonis-bull-queue#32]
We trying to use bull-queue after init adonis with "API Starter Kit", but there is an error when we try to run `node ace configure @rlanz/bull-queue`.

```
node -v
# v21.6.1
```

**How to reproduce:**
`npm init adonisjs@latest hello-world`

Select "API Starter Kit".

Run:
```
npm install @rlanz/bull-queue
# and
node ace configure @rlanz/bull-queue
```

Get this following error: `[ error ] Cannot configure module "@rlanz/bull-queue". The module does not export the configure hook`

**Comment by RomainLanz** [comment]
Hey @beletate! 👋🏻 

This package is not yet available for AdonisJS 6.

**Comment by jigs1996** [comment]
@RomainLanz Could you please, let us know if we will get an update to support AdonisJS v6?

**Comment by irshadahmad21** [comment]
@RomainLanz When can we expect an update for this package for Adonis V6?

**Comment by RomainLanz** [comment]
You can try the release `2.0.0-6` and let me know what you think!

**Comment by irshadahmad21** [comment]
> You can try the release `2.0.0-6` and let me know what you think!

Thank you. I will give it a try.

**Comment by QU35T-code** [comment]
> 2.0.0-6

It seems to work fine with this version

```bash
> node ace configure @rlanz/bull-queue

DONE:    create config/queue.ts
DONE:    update .env file
DONE:    update start/env.ts file
DONE:    update adonisrc.ts file
``

**Comment by RomainLanz** [comment]
Cool, I will move the beta release to `3.0.0-1` later today because I plan to release another version for AdonisJS 5 that has some breaking changes.

Keep testing and let me know if you find anything!

**Comment by QU35T-code** [comment]
> Cool, I will move the beta release to `3.0.0-1` later today because I plan to release another version for AdonisJS 5 that has some breaking changes.
> 
> Keep testing and let me know if you find anything!

I've just realized that I can't access to the `Queue` module and call the `dispatch` method ? Am I doing something wrong ?

![image](https://github.com/RomainLanz/adonis-bull-queue/assets/51704860/f05a0c78-4aa3-4a81-a290-7a983519a5f6)

**Comment by RomainLanz** [comment]
It has to be `import queue from '@rlanz/bull-queue/services/main';`

**Comment by QU35T-code** [comment]
Thanks !! I just I didn't look at the documentation for the right branch :/

![image](https://github.com/RomainLanz/adonis-bull-queue/assets/51704860/cfc995a6-3feb-4fcb-a04b-921ee92ca146)

**Comment by RomainLanz** [comment]
On a side note, you must install the version `3.0.0-1` for AdonisJS 6 now. I push a breaking update for the AdonisJS 5 releases that took the `2.0.0` version. 👍🏻

**Comment by RomainLanz** [comment]
Officially released as `3.0.0`!
